src/panels/iframe/ha-panel-iframe.ts

```diff
@@ -41,7 +41,9 @@ class HaPanelIframe extends LitElement {
           )}
           src=${this.panel.config.url}
           sandbox="allow-forms allow-popups allow-pointer-lock allow-same-origin allow-scripts allow-modals allow-downloads"
-          allow="fullscreen"
+          allow=${ifDefined(
+            this.panel.config.allow === null ? 'fullscreen' : this.panel.config.allow
```

Comment:
`config.allow` will not be `null` but `undefined`

Reply:
```suggestion
            this.panel.config.allow ?? 'fullscreen'
```